src/gateway/requests-from-cache.js

```diff
@@ -105,7 +110,7 @@ export const asCachedRequest = (
             guttedResponse[FROM_CACHE_PROP_NAME] = FRESHLY_PRUNED;
             return guttedResponse;
         })
-        .buffer(buffer);
+        .buffer(true);
```

Comment:
We could pass nothing here; that is also equivalent to `true`, but I'd rather be explicit.